index.html

```diff
@@ -14,6 +14,12 @@
     <meta name="theme-color" content="#000" />
 
     <!--  fonts imports  -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap"
```

Comment:
I don't think we need all of those variants:
- italic style isn't used for any texts
- we only use weights `600` and `700`, we import much more here